tests/mocha/block_test.js

```diff
@@ -324,16 +325,6 @@ suite('Blocks', function() {
             }
           ]
         },
-        {
-          "type": "statement_block",
-          "message0": "%1",
-          "args0": [
-            {
-              "type": "input_statement",
-              "name": "STATEMENT"
-            }
-          ]
-        },
       ]);
     });
     teardown(function() {
```

Comment:
This teardown needs to not remove statement block.

Reply:
Done